types/node/http.d.ts

```diff
@@ -621,6 +621,15 @@ declare module 'http' {
          * @since v14.5.0, v12.19.0
          */
         protocol: string;
+        /**
+         * Whether the request is send through a reused socket.
+         * @since v13.0.0, v12.16.0
+         */
+        reusedSocket: boolean;
+        /**
+         * Limits maximum response headers count. If set to 0, no limit will be applied. Default = 2000.
```

Comment:
Potentially we could use the [`@default`](https://jsdoc.app/tags-default.html) tag here?

```suggestion
         * Limits maximum response headers count. If set to 0, no limit will be applied.
         * @default 2000
```